<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainStart</class>
 <widget class="QDialog" name="MainStart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1207</width>
     <height>628</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1191</width>
      <height>968</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QFrame" name="frame">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>950</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1200</width>
          <height>950</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="EffectsButtonPage2_2">
        <property name="geometry">
         <rect>
          <x>471</x>
          <y>68</y>
          <width>81</width>
          <height>50</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="ExitButton_2">
        <property name="geometry">
         <rect>
          <x>1045</x>
          <y>68</y>
          <width>61</width>
          <height>50</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="AboutUsButtonPage2_2">
        <property name="geometry">
         <rect>
          <x>682</x>
          <y>68</y>
          <width>81</width>
          <height>50</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="BackButton_3">
        <property name="geometry">
         <rect>
          <x>55</x>
          <y>62</y>
          <width>201</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="upload_image">
        <property name="geometry">
         <rect>
          <x>1085</x>
          <y>375</y>
          <width>91</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="SaveButton">
        <property name="geometry">
         <rect>
          <x>1120</x>
          <y>510</y>
          <width>61</width>
          <height>50</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_pic">
        <property name="geometry">
         <rect>
          <x>103</x>
          <y>194</y>
          <width>469</width>
          <height>421</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="filter_1">
        <property name="geometry">
         <rect>
          <x>73</x>
          <y>795</y>
          <width>90</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="filter_2">
        <property name="geometry">
         <rect>
          <x>183</x>
          <y>795</y>
          <width>90</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="filter_4">
        <property name="geometry">
         <rect>
          <x>403</x>
          <y>795</y>
          <width>90</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="filter_3">
        <property name="geometry">
         <rect>
          <x>293</x>
          <y>795</y>
          <width>90</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="filter_5">
        <property name="geometry">
         <rect>
          <x>513</x>
          <y>795</y>
          <width>90</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="filter_6">
        <property name="geometry">
         <rect>
          <x>623</x>
          <y>795</y>
          <width>90</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="filter_9">
        <property name="geometry">
         <rect>
          <x>949</x>
          <y>795</y>
          <width>90</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="filter_8">
        <property name="geometry">
         <rect>
          <x>839</x>
          <y>795</y>
          <width>90</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="filter_11">
        <property name="geometry">
         <rect>
          <x>1061</x>
          <y>845</y>
          <width>90</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="filter_7">
        <property name="geometry">
         <rect>
          <x>729</x>
          <y>795</y>
          <width>90</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="filter_12">
        <property name="geometry">
         <rect>
          <x>517</x>
          <y>890</y>
          <width>90</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="filter_10">
        <property name="geometry">
         <rect>
          <x>1059</x>
          <y>795</y>
          <width>90</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="filter_21">
        <property name="geometry">
         <rect>
          <x>75</x>
          <y>845</y>
          <width>90</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="filter_18">
        <property name="geometry">
         <rect>
          <x>951</x>
          <y>845</y>
          <width>90</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="filter_22">
        <property name="geometry">
         <rect>
          <x>407</x>
          <y>890</y>
          <width>90</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="filter_20">
        <property name="geometry">
         <rect>
          <x>185</x>
          <y>845</y>
          <width>90</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="filter_14">
        <property name="geometry">
         <rect>
          <x>515</x>
          <y>845</y>
          <width>90</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="filter_13">
        <property name="geometry">
         <rect>
          <x>405</x>
          <y>845</y>
          <width>90</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="filter_16">
        <property name="geometry">
         <rect>
          <x>731</x>
          <y>845</y>
          <width>90</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="filter_17">
        <property name="geometry">
         <rect>
          <x>841</x>
          <y>845</y>
          <width>90</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="filter_15">
        <property name="geometry">
         <rect>
          <x>625</x>
          <y>845</y>
          <width>90</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="reset">
        <property name="geometry">
         <rect>
          <x>1110</x>
          <y>280</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_pic_2">
        <property name="geometry">
         <rect>
          <x>599</x>
          <y>194</y>
          <width>469</width>
          <height>421</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="choose_image">
        <property name="geometry">
         <rect>
          <x>126</x>
          <y>708</y>
          <width>401</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Choose image</string>
        </property>
       </widget>
       <widget class="QPushButton" name="merge">
        <property name="geometry">
         <rect>
          <x>646</x>
          <y>708</y>
          <width>401</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Merge</string>
        </property>
       </widget>
       <widget class="QPushButton" name="flipH">
        <property name="geometry">
         <rect>
          <x>646</x>
          <y>708</y>
          <width>401</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Flip Horizontally</string>
        </property>
       </widget>
       <widget class="QPushButton" name="flipV">
        <property name="geometry">
         <rect>
          <x>126</x>
          <y>708</y>
          <width>401</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Flib Vertical</string>
        </property>
       </widget>
       <widget class="QPushButton" name="filter_19">
        <property name="geometry">
         <rect>
          <x>295</x>
          <y>845</y>
          <width>90</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="filter_23">
        <property name="geometry">
         <rect>
          <x>625</x>
          <y>890</y>
          <width>90</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="filter_24">
        <property name="geometry">
         <rect>
          <x>731</x>
          <y>890</y>
          <width>90</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <zorder>label</zorder>
       <zorder>filter_12</zorder>
       <zorder>filter_22</zorder>
       <zorder>EffectsButtonPage2_2</zorder>
       <zorder>ExitButton_2</zorder>
       <zorder>AboutUsButtonPage2_2</zorder>
       <zorder>BackButton_3</zorder>
       <zorder>upload_image</zorder>
       <zorder>SaveButton</zorder>
       <zorder>label_pic</zorder>
       <zorder>filter_1</zorder>
       <zorder>filter_2</zorder>
       <zorder>filter_4</zorder>
       <zorder>filter_3</zorder>
       <zorder>filter_5</zorder>
       <zorder>filter_6</zorder>
       <zorder>filter_9</zorder>
       <zorder>filter_8</zorder>
       <zorder>filter_11</zorder>
       <zorder>filter_7</zorder>
       <zorder>filter_10</zorder>
       <zorder>filter_21</zorder>
       <zorder>filter_18</zorder>
       <zorder>filter_20</zorder>
       <zorder>filter_14</zorder>
       <zorder>filter_13</zorder>
       <zorder>filter_16</zorder>
       <zorder>filter_17</zorder>
       <zorder>filter_15</zorder>
       <zorder>reset</zorder>
       <zorder>label_pic_2</zorder>
       <zorder>choose_image</zorder>
       <zorder>merge</zorder>
       <zorder>flipH</zorder>
       <zorder>flipV</zorder>
       <zorder>filter_19</zorder>
       <zorder>filter_23</zorder>
       <zorder>filter_24</zorder>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
